docs/csharp/programming-guide/concepts/linq/how-to-populate-an-xml-tree-with-an-xmlwriter-linq-to-xml.md

```diff
@@ -13,7 +13,7 @@ One way to populate an XML tree is to use <xref:System.Xml.Linq.XContainer.Creat
   
 ```csharp  
 string xslMarkup = @"<?xml version='1.0'?>  
-<xsl:stylesheet xmlns:xsl='http://www.w3.org/1999/XSL/Transform' version='1.0'>  
+<xsl:stylesheet xmlns:xsl='https://www.w3.org/1999/XSL/Transform' version='1.0'>  
```

Comment:
the idea was to update the links from the MarkDown content to begin with not the code. 
if I create a new XSLT file on VS, it creates with the namespace using http. I'd keep it that way.
```
<?xml version="1.0" encoding="utf-8"?>
<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
    xmlns:msxsl="urn:schemas-microsoft-com:xslt" exclude-result-prefixes="msxsl"
>
```